en/docs/learn/examples/inbound-endpoint-examples/inbound-endpoint-http-protocol.md

```diff
@@ -69,7 +68,7 @@ Accept-Encoding: gzip,deflate
 Content-Type: text/xml;charset=UTF-8
 SOAPAction: "urn:getQuote"
 Content-Length: 492
-Host: localhost:8290
+Host: localhost:8085
```

Comment:
Replaced the port here with the port in the request URL since the two should be the same.
Having the Host port 8290 caused the below error:

![image](https://github.com/wso2/docs-mi/assets/47134017/ea03ed2e-293a-45c4-8d43-074a6391ac82)